storage/src/quickstart.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.storage
+
+/**
+ * A class which instantiates a Cloud Storage client and lists all objects in a
+ * bucket.
+ */
+
+import com.google.cloud.storage.StorageOptions
+
+fun quickstart(bucketName: String) {
```

Comment:
What quickstart does is identical to "info" command

Reply:
Yes, this is intentional. The `quickstart` sample should be an "all in one" sample. The `info` command now calls the `quickstart` method, so the code is still DRY

Reply:
BTW the reason I created the `quickstart` is so we can have a more coherent sample for the landing page. All our other languages have a `quickstart` example which you can copy-paste and it works pretty much immediately.